doctor/views.py

```diff
@@ -123,7 +108,7 @@ def extract_doc_content(request) -> Union[JsonResponse, HttpResponse]:
 
     # Get page count if you can
     page_count = get_page_count(fp, extension)
-    os.remove(form.cleaned_data["fp"])
+    cleanup_form(form)
```

Comment:
this makes me wonder...

Reply:
`cleanup_form` does the same thing as `os.remove...`, so no change in functionality here, I think.